SwiftPackage/Sources/BridgeClientExtension/Data Archiving/JsonResultArchiveBuilder.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  ArchiveBuilder.swift
+//  
+//
+
+import Foundation
+import JsonModel
+import ResultModel
+
+/// A simple archive builder to use when uploading a single json file that may not
+/// implement the `AssessmentResult` protocol.
+open class JsonResultArchiveBuilder : ResultArchiveBuilder {
+    
+    public let uuid: UUID = .init()
+    public let json: Data
+    public let fileInfo: FileInfo
+    public let startedOn: Date
+    
+    public var endedOn: Date { fileInfo.timestamp }
+    public var adherenceData: JsonSerializable? { nil }
+    
+    /// The archive that backs this builder.
+    let archive: StudyDataUploadArchive
+    
+    public var identifier: String {
+        archive.identifier
+    }
+    
+    @available(*, deprecated, message: "Bridge Exporter V1 no longer supported - schema identifier and revision are ignored.")
```

Comment:
```suggestion
    @available(*, deprecated, message: "Bridge Exporter V2 no longer supported - schema identifier and revision are ignored.")
```
Exporter v1 as far as I recall hasn't been used/supported since the initial batch of ResearchKit/AppCore apps. We're on V3 now...